bindings/python/malamute/_malamute_ctypes.py

```diff
@@ -7,7 +7,7 @@
 import os, sys
 from ctypes import *
 from ctypes.util import find_library
-import czmq
+import czmq.git
```

Comment:
@vyskocilm @sappo this is what was happening to me! I'm pretty sure this shouldn't happen